### DISCUSSION TITLE
Can the Comunica web service be queried using the SPARQL 1.1 Protocol?

### DISCUSSION THREAD

**Discussion by wouterbeek** [comunica/comunica#743] [Q&A] [answered]
#### Issue type:

- :question: Question 

____
#### Question:

Is it possible for clients to send SPARQL 1.1 Protocol query requests to the Comunica web service?  For example through the cURL command-line tool.

- Example of a SPARQL query in the Comunica GUI: [link](https://query.linkeddatafragments.org/#datasources=https%3A%2F%2Fapi.triplydb.com%2Fdatasets%2Fwikimedia%2Fwikidata%2Ffragments&query=PREFIX%20wdt%3A%20%3Chttp%3A%2F%2Fwww.wikidata.org%2Fprop%2Fdirect%2F%3E%0APREFIX%20wd%3A%20%3Chttp%3A%2F%2Fwww.wikidata.org%2Fentity%2F%3E%0ASELECT%20DISTINCT%20*%20WHERE%20%7B%0A%20%20%3Fs%20wdt%3AP3000%20%3Fvalue.%0A%20%20%3Fs%20wdt%3AP984%20%22SVK%22.%0A%7D%0A)
- SPARQL 1.1 Protocol specification: [link](https://www.w3.org/TR/sparql11-protocol/)

**Comment by rubensworks** [comment]
Someone will answer your question soon. In the meantime, you might be able to get help more quickly on our [Gitter channel](https://gitter.im/comunica/Lobby).

**Comment by rubensworks** [comment, marked as answer]
Yes, that's possible, but requires the use of the `comunica-sparql-http` tool: https://comunica.dev/docs/query/getting_started/setup_endpoint/

**Comment by wouterbeek** [comment]
Thanks for the info, will check that option out.
